### ISSUE TITLE
Management Page Message Field Discrepancies

### ISSUE THREAD

**Issue by small-fork** [stcr/subscribe-to-comments-reloaded#677]
Hello,

I've noticed some discrepancies within the management page message fields. I made a screencast illustrating the issues: https://www.loom.com/share/fd40c57047d44d09b5bb573eade82798

A summary:

1. The "Subscription processed" field says it's used for when double-check is disabled. However, I have double check enabled and when a user clicks the link to confirm in their email, the message showed on the management page is the one in this field.
2. While the "Subscription processed (DCI)" says it's used if double-check is enabled (as it is for me), I never see it in the process. Also, the default text in the field says "In order to confirm your request, please check your email for the verification message and follow the instructions." which indicates it isn't what you see AFTER clicking the confirm link, which DOES seem to be the case for the "Subscription processed" field. I thought maybe this showed after submitting your comment, but that only says "Check your email to confirm your subscription" and doesn't seem editable, though I added on a sentence using the CSS ::after property
3.  The "Subscription processed" field says it accepts the shortcodes [post_title] & [post_permalink], however when I used those in the field, they didn't populate. (See screencast)

Thank you!

**Comment by BobaWebDev** [comment]
Hi @small-fork 

When `double check` functionality is enabled, it will show the `Subscription processed (DCI)` if that email address is not associated with an existing subscription and `Subscription processed` will show if that email already has a subscription active (on any post).

Is that maybe what's happening, did the email address you used already have an active subscription?

I just made a new comment on there, but since emailing is not enabled on the site, can you send over the confirmation URL (don't click it). Want to check it out for both the message and the shortcodes.

**Comment by small-fork** [comment]
@BobaWebDev No, these were new subscriptions, otherwise they'd not receive a confirmation email in the first place. 

I'm still confused about the face that the "Subscription processed (DCI)" message says you have to STILL confirm, but the "Subscription processed" says it's BEEN confirmed. If the Subscription processed is not for SCI, why would confirmation even be a part of it? And for the Subscription processed (DCI), why would it be telling you you NEED to confirm if it's a message saying it's been processed? 

With NEW subscribers, what I see when clicking the confirmation link is the text I have in the "Subscription processed" field. I never see the "Subscription processed (DCI)" message. 

I altered the messages so that at the beginning of each I say which one it is, so you'll know. I also added the shortcodes back in so you can see how they weren't populating.

Here is your confirmation link: https://wordpress-448298-2270282.cloudwaysapps.com/comment-subscriptions?srek=852feb9eac0c174fbcb62361f30041b5&srk=ee6959a354b478a57a2c39612777b5fd&srp=4420&sra=c&srsrc=e&confirmation_email=y&post_permalink=https://wordpress-448298-2270282.cloudwaysapps.com/ethics/vegans-in-the-renaissance-the-history-of-veganism-part-three/
